src/sass/modules/_m-footer.scss

```diff
@@ -19,7 +19,94 @@
   }
 }
 
-  .footer-inner {
+.feedback-widget {
+  padding: 1.5em 0;
+  background-color: $color-gray-lightest;
+  color: $color-gray-dark;
+
+  .row {
+    padding: 0 0.9375rem;;
+  }
+
+  .feedback-widget-title {
+    margin: 0 0 0.5em 0;
+  }
+
+  .feedback-button {
+    width: auto;
+    // margin-top: initial;
+  }
+
+  .feedback-widget-form {
+    order: 1;
+    flex: 0 0 100%;
+    // padding-right: 1em;
+  }
+
+  .feedback-widget-need-help {
+    order: 0;
+    flex: 0 1 100%;
+    .feedback-widget-need-help-inner {
+      background-color: $color-aqua-lightest;
+      padding: 1em;
+      margin: 1em 0;
+      h3 {
```

Comment:
I think I would use `>` or add a class to this `h3`. It's another one of those things that can become a pain later if someone makes changes in here later and wants to tweak the header styles.

Reply:
Yeah that makes sense, I'll try to get the style cleaned up in general